src/funfuzz/js/jsfunfuzz/gen-grammar.js

```diff
@@ -779,10 +925,18 @@ var exprMakers =
   function(d, b) { return cat([Random.index(leftUnaryOps), makeExpr(d, b)]); },
 
   // Methods
-  function(d, b) { var id = makeId(d, b); return cat(["/*UUV1*/", "(", id, ".", Random.index(allMethodNames), " = ", makeFunction(d, b), ")"]); },
-  function(d, b) { var id = makeId(d, b); return cat(["/*UUV2*/", "(", id, ".", Random.index(allMethodNames), " = ", id, ".", Random.index(allMethodNames), ")"]); },
-  function(d, b) { return cat([makeExpr(d, b), ".", Random.index(allMethodNames), "(", makeActualArgList(d, b), ")"]); },
-  function(d, b) { return cat([makeExpr(d, b), ".", "valueOf", "(", uneval("number"), ")"]); },
+  function(d, b) { var id = makeId(d, b); return cat([
```

Comment:
Minor nit but template literals would make all of these methods much cleaner.

Reply:
Template literals are on the horizon...